tasks/bootstrap.yml

```diff
@@ -48,7 +48,7 @@
   with_items: "{{ sb_debian_base_extra_packages }}"
 
 - include: bootstrap-firewall.yml
-  when: sb_debian_base_firewall is defined
+  when: sb_debian_base_firewall == 'yes'
```

Comment:
I wonder use `true` instead of `"yes"`, sometimes I use true instead of yes, I wonder if we could do `when: sb_debian_base_firewall == true` so we could use both `"yes"` and `true` or even we could do just `when: sb_debian_base_firewall`  and this variable should be under defaults like `sb_debian_base_firewall: true` or `yes`